smartsim/ml/data.py

```diff
@@ -108,12 +108,13 @@ def download(self, client: Client) -> None:
         """
         try:
             info_ds = client.get_dataset(self._ds_name)
-        except RedisReplyError:
+        except RedisReplyError as e:
             # If the info was not published, proceed with default parameters
             logger.warning(
                 "Could not retrieve data for DataInfo object, the following "
                 "values will be kept."
             )
+            logger.debug(f"Original error from Redis was {e}")
```

Comment:
I don't think the original error should be a debug level message. An exception should always be logged, shouldn't it?

Reply:
Promoted to `error`.